aii-pxelinux/src/main/perl/pxelinux.pm

```diff
@@ -4,12 +4,13 @@ use Sys::Hostname;
 use CAF::FileWriter;
 use CAF::Object;
 use NCM::Component::ks qw (ksuserhooks);
+use AII::Shellfe qw(:pxe_constants);
```

Comment:
I am not sure to understand my previous answer! The choice made was to have all the constants related to `pxelinux` options defined here as some are set by `aii-shellfe` configuration file or command line options. The idea was not to have half of them defined in `Shellfe.pm` and the other half in `pxelinux.pm` and also to avoid a circular dependency where `pxelinux.pm` needs to "use" `AII::Shellfe` and `Shellfe` needs to "use" `NCM::Component::pxelinux`. But I am open to suggestion if you don't think this is appropriate...